### ISSUE TITLE
Add support for nn_batch_norm3d

### ISSUE THREAD

**Issue by mattwarkentin** [mlverse/torch#353]
Hi @dfalbel,

Would you be interested in a simple PR to add support for 3D batchnorm, in line with https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm3d.html#torch.nn.BatchNorm3d?

**Comment by dfalbel** [comment]
Sure! Would be great!!
